### PR TITLE
Add HTML example for <a> (issue #1014)

### PR DESCRIPTION
**Pull request by stephanmax**
This example includes external, relative, email, and telephone links.

### CODE REVIEW

File: live-examples/html-examples/inline-text-semantics/css/a.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+a[href^="http"]::before {
+  content: '↑ ';
+  font-style: italic;
+}
```
Comment (schalkneethling) [inline live-examples/html-examples/inline-text-semantics/css/a.css]: Nit: Missing empty line at end of file.

File: live-examples/html-examples/inline-text-semantics/css/a.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+a[href^="http"]::before {
+  content: '↑ ';
```
Comment (schalkneethling) [inline live-examples/html-examples/inline-text-semantics/css/a.css]: I would suggest using the following icon(as SVG) to stay consistent with the rest of MDN:
https://fontawesome.com/icons/external-link-alt?style=solid

**Comment by stephanmax** [comment]
Good stuff, @schalkneethling. Changes are in.

File: live-examples/html-examples/inline-text-semantics/css/a.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+a[href^="http"]::before {
+  background: url(/media/examples/external-link-alt-solid.svg) no-repeat;
+  content: '';
+  display: inline-block;
+  margin-right: 3px;
+  opacity: .6;
+  width: 0.9em;
+  height: 0.8em;
```
Comment (schalkneethling) [inline live-examples/html-examples/inline-text-semantics/css/a.css]: Nit: Code Style: No leading zero for fraction based values.
https://stylelint.io/user-guide/rules/number-leading-zero/

**Comment by stephanmax** [comment]
Thanks, @schalkneethling. I updated the CSS.
Should we include the example’s CSS in the `lint-css` npm script for the future?

**Comment by schalkneethling** [comment]
@stephanmax Yup, that is something I am working on.

**Pull request merged**